README.md

```diff
@@ -25,67 +27,83 @@ Upon deploying the Auction Contract you need to supply the `contract package has
 - `"reserve_price"`: U512 type of reserve price aka smallest permitted selling price.
 - `"token_id"`: String/Text id of the NFT token put up for auction.
 - `"start_time"`: u64 UNIX timestamp of auctions starting time.
-- `"cancellation_time"`:  u64 UNIX timestamp of the latest time bids on the auction can be cancelled.
-- `"end_time"`:  u64 UNIX timestamp of the time the auction will end.
+- `"cancellation_time"`: u64 UNIX timestamp of the latest time bids on the auction can be cancelled.
+- `"end_time"`: u64 UNIX timestamp of the time the auction will end.
 
 ## Usage
+
 The three possible endpoints of the contract are:
+
 - `"bid"`: self-explanatory.
-- `"cancel_bid"`: anyone can cancel their bid in an english auction until the cancellation deadline. Not usable with dutch auctions, as with those the first valid bid wins immidiately. 
+- `"cancel_bid"`: anyone can cancel their bid in an english auction until the cancellation deadline. Not usable with dutch auctions, as with those the first valid bid wins immidiately.
 - `"finalize"`: called after the end of an english auction to finish up the auction the distribute motes and token according to result. Dutch auctions finalize themselves upon the first valid bid.
 
 Dutch auctions will end immidiately when a valid bid has been made.
 English auctions will run their course and must be finalized after the end time to distribute motes and the NFT token.
 
-When the auction end 
+When the auction end
 a) there were no winning bids or an error has occured, every bid will be returned to the bidder, and the NFT token to the owner.
 b) the auction was won by a bidder and as such they will receive the NFT token. Bids that did not win will be returned to their respective bidders.
 The winning bid will be sliced into bid/1000 pieces and these pieces will be forwarded to those listed in the NFT tokens commission meta data. All remaining motes will be sent to the beneficiary account provided by the auctions deployer.
 
 ## Make commands
+
 ### prepare
+
 Adds wasm to the cargo compilation targets.
 
 ### build-contract
+
 Builds the contracts using the nightly toolchain with wasm as the compilation target.
 
 ### test-only
+
 Run all tests inside the workspace.
 
 ### copy-wasm-file-to-test
+
 Copies the `.wasm` files into `/tests/wasm` folder, where the test framework is set to look for them.
 
 ### test
+
 Executes the `build-contract` and `copy-wasm-file-to-test`, then `test-only` commands.
 
 ### clippy
+
 Executes the clippy linter on the contract and test codes.
 
 ### lint
+
 Runs `clippy` and `format`
 
 ### check-lint
+
 Runs `clippy` then executes `fmt` with `--check` enabled. Gives errors for clippy warnings.
 
 ### format
+
 Applies formatting to the codes.
 
 ### clean
+
 Artifact removal command. (`.wasm` files, `target` folders)
 
 ## Rust version
+
```

Comment:
Update as appropriate

Reply:
Updated to `rustc 1.68.0-nightly (0468a00ae 2022-12-17)`